### DISCUSSION TITLE
Can I copy a file directly from draw.io

### DISCUSSION THREAD

**Discussion by qwerty3858** [jgraph/drawio#4018] [General]
Can I copy the extensions.min.js code in draw.io directly to my own project code and use it? I want to use this extensions.min.js for mermaid

**Comment by davidjgraph** [comment]
Yes, that's allowed under the Apache 2.0.

**Reply by qwerty3858** [reply]
While I was copy extensions.min.js , do we need to mention it somewhere as a reference in my own project for copyright?

and Also, we will use it in some business that will make money, is this also allowed?

**Reply by qwerty3858** [reply]
Is there extensions.min.js open source code? just for mermaid without unminify like extensions.js.

**Reply by davidjgraph** [reply]
We'll have a look at adding the mermaid code into the public build. Technically, there shouldn't be a problem with that, technically...

**Reply by davidjgraph** [reply]
https://github.com/jgraph/drawio/discussions/3805
